test/spec/modules/stroeerCoreBidAdapter_spec.js

```diff
@@ -568,6 +585,20 @@ describe('stroeerCore bid adapter', function() {
           }],
           'ver': {},
           'user': {
+            'eids': [
```

Comment:
Do the same thing like what we did with euids:

```
'euids': userIds
```

So do exactly this instead:
```
'eids': userIdAsEids
```

The value will be immutable if you use Object.freeze as well (see above comment). So can do with confidence.